### PR TITLE
Feature/#34/create graphq l types and queries for main resource news and its dependant resources

### PR DESCRIPTION
**Pull request by acousma82**
Update NewsItem Model

* add url to news item because avery news item has a source url
* add address to news_item because every news_item can have an address

#34

Add resource NewsItem to graphql schema

* graphql api needs to provide news items
*  to implemt feature:
    * add news_item  and connected  content_block resources to graphql as types
    * add query to return for one news item by id
    * add query to return for all news items

#34

Correct bug in news_item model

* news item has duplicate has_one association with address. this commit will remove one of them

#34

Add News Items to seeds and recreate database

#34

Fix Bugs to make graphql server work

* file content_block_type needs class name ContentBlockType
*  file name adress_type -> address_type

#34

 Add new query types

* In  issue #36 new queries were requested
* this commit will  add new query types to limit the amount of data for all resources and order the queried data

#34

### CODE REVIEW

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -27,5 +30,13 @@ def all_event_records
...
       EventRecord.find(id)
     end
+
+    def all_news_items
+      NewsItem.all
+    end
+
+    def news_item(id:)
+      NewsItem.find(id)
+    end
```
Comment (marcometz) [inline app/graphql/types/query_type.rb]: ich bleibe dabei, das ich mit dem Naming der `QueryTypes` nicht glücklich bin und habe mittlerweile auch einige BestPractices dazu gefunden.

- https://graphql.github.io/learn/thinking-in-graphs/
- https://mattboldt.com/2019/01/07/rails-and-graphql/
- https://developers.facebook.com/docs/graph-api/reference/

Comment (acousma82) [reply app/graphql/types/query_type.rb]: Ok, dann ändere ich es.

Comment (acousma82) [reply app/graphql/types/query_type.rb]: Im rails graphql tutorial, war es eben aber anders und davon bin ich ausgegangen

Comment (acousma82) [reply app/graphql/types/query_type.rb]: Habe direkt ein issue dazu gemacht: #38

**Pull request merged**